### ISSUE TITLE
can do anything

### ISSUE THREAD

**Issue by sophienix** [jrbudda/Vivecraft_112#182]
i can do anything when the game has started my hands are in the floor and i cant move i also have ovr tool kit open and its working fine its just minecraft vr it did this even before i had ovr tool kit. how do i fix this? i even used the latest verson and im using quest 1 for vr headset.

**Comment by Techjar** [comment]
Make sure the Steam client is running, then go into the bindings and ensure they are set to default.

**Comment by sophienix** [comment]
its actin like it cant see my vr controlers

**Comment by sophienix** [comment]
i should of said that in the begining

**Comment by Techjar** [comment]
Yeah I understood. Refer to my previous comment.

**Comment by sophienix** [comment]
ok

**Comment by sophienix** [comment]
it is open but i  cant find it i found it in sream vr but it says open steam to edit when i have done it in steam vr

**Comment by Techjar** [comment]
That means you don't have Steam running, which I explicitly mentioned.

**Comment by sophienix** [comment]
i do have it running thats how i opened steam vr

**Comment by Techjar** [comment]
Restart it then, it has alzheimer's sometimes.

**Comment by sophienix** [comment]
i can even see my steam games in steam vr home

**Comment by sophienix** [comment]
ok

**Comment by sophienix** [comment]
the controls are set to default

**Comment by sophienix** [comment]
let me try again

**Comment by sophienix** [comment]
it works all i had to do was restart

**Comment by Techjar** [comment]
👍
